task/10-katas-1-tasks.js

```diff
@@ -113,7 +186,33 @@ function getZigZagMatrix(n) {
  *
  */
 function canDominoesMakeRow(dominoes) {
-    throw new Error('Not implemented');
+    function task10(current, value, left) {
+        if (left == 0) {
```

Comment:
operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)